docs/UserGuide.md

```diff
@@ -282,6 +282,11 @@ Format: `focus INDEX`
 
 * Additional details of the Candidate will be displayed on the center panel in the application.
 * To switch to another Candidate's information, user will just need to type the new command and the new index of the candidate.
```

Comment:
Do you think we should change the new command to `focus` command instead? Users might take the word new as another command.